.github/workflows/build-latest.yaml

```diff
@@ -0,0 +1,23 @@
+name: Built latest
+
+on:
+  push:
+    branches: [ master ]
+
```

Comment:
add multiple triggers in a single config file rather than having two identical ones